docs/using.md

```diff
@@ -323,40 +226,69 @@ authenticated users. The following procedure describes how to mount this folder.
-However, MiniShift is configured to persist files stored under the following host directories:
-
-* `/data`
-* `/var/lib/minishift`
-* `/var/lib/docker`
-
-Here is an example PersistentVolume config to persist data in the `/data` directory:
-
-```yaml
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: pv0001
-spec:
-  accessModes:
-    - ReadWriteOnce
-  capacity:
-    storage: 5Gi
-  hostPath:
-    path: /data/pv0001/
+<a name="interacting-with-openshift"></a>
+## Interacting with OpenShift
+
+<a name="openshift-client-binary-oc"></a>
+### OpenShift client binary (oc)
+
+The `minishift start` command creates an OpenShift instance using the
+[cluster up](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) approach.
+
+For this purpose it copies the `oc` binary onto  your host. You find it under
+`~/.minishift/cache/oc/\<OpenShift version\>/oc`. You can add this binary to your `PATH` variable
+in order to use `oc`, for example:
+
+    $ export PATH=$PATH:~/.minishift/cache/oc/v1.4.1
+
+In future versions we will provide a command to assist in setting up the `PATH`.
+See Github issue [#142](https://github.com/minishift/minishift/issues/142).
+
+For an introduction to `oc` usage, refer to the [Get Started with the CLI](https://docs.openshift.com/enterprise/3.2/cli_reference/get_started_cli.html)
+section in the OpenShift documentation.
+
+<a name="login"></a>
+### Login
+
+Per default _cluster up_ uses an [AllowAllPasswordIdentityProvider](https://docs.openshift.org/latest/install_config/configuring_authentication.html#AllowAllPasswordIdentityProvider)
+for authentication against the local cluster. This means any non-empty username and password can
+be used to login to the local cluster. The recommended username and password are
+developer/developer, since it also has a default project _myproject_ set up.
+
+To login as administrator, use the system account:
+
+```shell
+$ oc login -u system:admin
+```
+
+In this case [client certificates](https://docs.openshift.com/enterprise/3.2/architecture/additional_concepts/authentication.html#api-authentication)
+are used which are stored in `~/.kube/config`. _cluster up_ will install
+the appropriate certificates as part of the bootstrap.
+
+**Note:** If you type `oc login -u system -p admin`, you will get logged in, but not as an administrator,
+but rather as an unprivileged user with no particular rights.
+
+To view the currently available login contexts, run:
+
+```
+$ oc config view
+```
+
+<a name="console"></a>
+### Console
+
+To access the [OpenShift console](https://docs.openshift.org/latest/architecture/infrastructure_components/web_console.html),
+run this command in a shell after starting Minishift to get the address:
+
+```shell
+$ minishift console
 ```
 
-<a name="private-container-registries"></a>
-## Private container registries
+<a name="services"></a>
+### Services
```

Comment:
That's OpenShift, even though this is broken atm. See issue #342. Hopefully by the end of the sprint we can correct the docs.